kits/vehicle-service-advisor/apps/.env.example

```
@@ -0,0 +1,4 @@
+VEHICLE_SERVICE_ADVISOR_FLOW_ID=
+LAMATIC_API_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_API_URL key should go before the VEHICLE_SERVICE_ADVISOR_FLOW_ID key

(UnorderedKey)